src/requests/autosuggest-selection.ts

```diff
@@ -8,6 +9,9 @@ export const autosuggestSelection = (
   options: AutosuggestOptions = {},
   sourceApi: 'text' | 'voice' = 'text',
 ): Promise<null> => {
+  const W3W_DOMAIN_REGEX = /^[\w.-_]*.(what3words.com|w3w.io){1}/ig
```

Comment:
What about the w3w.co domain, is it relevant here?

Reply:
No it isn't included here as you cannot make a request to the API within w3w.co domain so it should do nothing also if the domain is set to w3w.co

Reply:
Good shout though!

Reply:
Also is the hyphen in the regex meant to be acting as a range or just the hyphen character itself? Just asking because this is a hot topic with the 3wa regex at the moment!

Reply:
Good spot... It's meant to be an explicit hyphen! Let me review the regex as I'm not even sure we care about anything before the what3words.com or w3w.io TLD regex.